### DISCUSSION TITLE
Plans for iOS

### DISCUSSION THREAD

**Discussion by Xiphe** [leo-benz/budgetbudget-native#9] [Ideas]
Hey Leo,

You write that you're planning to bring bb to iOS just without the MM sync. I'm interested about the scope of that app. Would that be more a companion app or a standalone budgeting solution?

I have an app called CashCash in my head that would more or less be mobile only and help keeping track of cash expenses (you're on the go, buy something cash and can make a note in CC right away). CC would sync its expenses to iCloud and then we'd have some sort of integration to import those to MM and thereby back to BB.

Does this come anywhere near to what you're planning?

Cheers 🖖

**Comment by leo-benz** [comment]
My current plan is to bring it to iOS (mainly iPadOS) with almost the same features and UI as the macOS app. My use-case is that I am often away from my Mac but still have my iPad with me. On this occasions I want to be able to look at my current budget and budget for the next month(s). I wanted to sync the budget data via iCloud so that I have read/write access from both devices. The MoneyMoney sync then only takes place when using the Mac app and the iOS version will work with the latest cached data synced via iCloud.

Your idea sounds great (even tough I rarely use cash nowadays) if I use it my manual MoneyMoney account is always outdated. It sounds like a great feature for an iPhone (or even Apple Watch) version. While I certainly could imagine it as a standalone app, I think the existing BudgetBudget Mac App could make the experience more seamless rather than needing an additional companion app.

**Comment by Xiphe** [comment]
That definitely sounds interesting! Thanks for sharing!
